docs/examples/linux-bridge-vlan.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: nmstate.io/v1beta1
```

Comment:
Could we add it to the list in the README?

Reply:
Done